ingest/indexer/service/indexer_streaming_service.go

```diff
@@ -240,6 +245,43 @@ func (s *indexerStreamingService) publishTxn(ctx context.Context, req types.Requ
 	return s.client.PublishTransaction(sdkCtx, txn)
 }
 
+// addTokenLiquidity adds the token liquidity to the event.
```

Comment:
```suggestion
// AddTokenLiquidity adds the token liquidity to the event.
```

Reply:
Will keep it as small letter as this will not be exported by using export_test.go